src/components/achievementCard/AchievementCard.js

```diff
@@ -14,7 +14,11 @@ export default function AchievementCard({cardInfo, isDark}) {
   return (
     <div className={isDark ? "dark-mode certificate-card" : "certificate-card"}>
       <div className="certificate-image-div">
-        <img src={cardInfo.image} alt="PWA" className="card-image"></img>
+        <img
+          src={cardInfo.image}
+          alt={cardInfo.imageAlt}
```

Comment:
Could you add a basic fallback alt text just in case someone doesn't configure it?
Like: `alt={cardInfo.imageAlt || "card thumbnail"}`

Reply:
I don't think you need a default. 
If you did not provide an `imageAlt` it will just not pass an `alt` attribute to an `<img>` at all, which looks right to me.

Reply:
@kartikcho Actually, I think you are right - it's better to have a default value for the alt. I just updated the PR.